doc/CC2ListView/convert.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+sed 's/||/\t/g' $1 > ${1/dsv/tsv}
+gawk -f doc/CC2ListView/convertTable.awk ${1/dsv/tsv}
```

Comment:
Please add GNU Awk to the list of required programs, both in the README and doc/newsboat.asciidoc.